CreateDB

```diff
@@ -30,14 +32,15 @@ CREATE TABLE Messages (
     created_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
-CREATE TABLE Multimedia (
+CREATE TABLE Photo (
     image BYTEA NOT NULL,
-    mid INTEGER REFERENCES Messages(mid) UNIQUE
+    mid INTEGER REFERENCES Messages(mid) PRIMARY KEY
```

Comment:
Wait a minute what is going on here?

Reply:
Me estoy dejando llevar por el powerpoint de ER to schema. Each table does have to have a primary key, be it a unique column or a composite key.

Reply:
El primary key de esta tabla puede ser el foreign key de otra porque es one to one. No se va a repetir. It's the same thing que hace django cuando haces inheritance de tablas, el primary key del child es el foreign key que apunta al parent